.eslintrc.json

```diff
@@ -1,5 +1,5 @@
 {
-    "extends": ["eslint:recommended", "google"],
+    "extends": ["eslint:recommended", "google", "prettier"],
```

Comment:
As i remember, we decided to move prettier config to another PR

Reply:
These commits come from merging prettier into my branch with tasks